### PR TITLE
Fix manual url prefixing

### PR DESCRIPTION
**Pull request by pradithya**
**What this PR does / why we need it**:

Fix manual URL prefixing which cause badly formatted url when the scheme is not http. This implementation will fallback to use `http://` scheme when it's not provided. 

**Does this PR introduce a user-facing change?**:

**Checklist**

- [X] Added unit test, integration, and/or e2e tests
- [X] Tested locally
- [N.A.] Updated documentation
- [N.A.] Update Swagger spec if the PR introduce API changes
- [N.A.] Regenerated Golang and Python client if the PR introduce API changes

### CODE REVIEW

**Comment by pradithya** [comment]
Looks like it breaks some e2e test. Let me fix it first.

File: api/pkg/transformer/server/rest/server.go [hunk 1]
```diff
@@ -358,3 +358,13 @@ func getHeaders(headers http.Header) map[string]string {
 	}
 	return resultHeaders
 }
+
+// getUrl return url or add default http scheme if scheme is not specified
+func getUrl(rawUrl string) (string, error) {
```
Comment (tiopramayudi) [inline api/pkg/transformer/server/rest/server.go:363]: why returning err if there is no chance it will throw error?

Comment (pradithya) [reply api/pkg/transformer/server/rest/server.go]: Fixed, I missed removing it after changing the implementation of `getUrl`

File: api/pkg/transformer/server/rest/server.go [hunk 2]
```diff
@@ -67,16 +67,13 @@ type hystrixHttpClient interface {
...
 	return NewWithHandler(o, nil, logger)
 }
 
 // New initializes a new Server with pipeline handler
-func NewWithHandler(o *config.Options, handler *pipeline.Handler, logger *zap.Logger) *HTTPServer {
-	predictURL := fmt.Sprintf("%s/v1/models/%s:predict", o.ModelPredictURL, o.ModelFullName)
-	if !strings.Contains(predictURL, "http://") {
-		predictURL = "http://" + predictURL
-	}
+func NewWithHandler(o *config.Options, handler *pipeline.Handler, logger *zap.Logger) (*HTTPServer, error) {
```
Comment (tiopramayudi) [inline api/pkg/transformer/server/rest/server.go:75]: this function also won't returning error

**Comment by tiopramayudi** [comment]
the rest is LGTM

**Review by ariefrahmansyah — APPROVED** [review]

**Pull request merged**